lighthouse-core/audits/critical-request-chains.js

```diff
@@ -30,6 +30,7 @@ class CriticalRequestChains extends Audit {
       name: 'critical-request-chains',
       description: 'Critical Request Chains',
       optimalValue: 0,
+      helpText: 'Analyze your app\'s critical request chains to learn how to load your app as quickly as possible. <a href="https://developers.google.com/web/tools/lighthouse/audits/critical-request-chains" rel="noreferrer noopener" target="lighthouse">Learn more</a>.',
       requiredArtifacts: ['networkRecords']
```

Comment:
:heavy_check_mark: I rewrote it to:

The Critical Request Chains below show you what resources are required for first render of this page. Improve page load by reducing the length of chains, reducing the download size of resources, or deferring the download of unnecessary resources.